srcpkgs/libXaw3dXft/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libXaw3dXft'
+pkgname=libXaw3dXft
+version=1.6.2h
+revision=1
+build_style=gnu-configure
+configure_args="--disable-static --enable-internationalization --enable-multiplane-bitmaps
+ --enable-gray-stipples --enable-arrow-scrollbars --with-pic --with-gnu-ld"
+hostmakedepends="automake libtool bison pkg-config"
+makedepends="libXft-devel libXmu-devel libXpm-devel xorg-util-macros"
+short_desc="This is an extended version of Xaw3d with support for UTF-8"
```

Comment:
```suggestion
short_desc="Extended version of Xaw3d with support for UTF-8"
```

The description should be terse. I know that lints complained about the article before, but simply removing it is enough to make the lint error go away.

Reply:
OK, I'll correct this as well.